src/p5.dance.js

```diff
@@ -455,6 +455,24 @@ module.exports = class DanceParty {
 
   // Dance Moves
 
+  alternateMoves(group, n, move1, move2) {
+    group = this.getGroupByName_(group);
+    let currentMeasure = Math.floor(this.getCurrentMeasure());
+    if (currentMeasure === 0) {
+      currentMeasure = 1;
+    }
+    group.forEach(sprite => {
+      this.changeMoveLR(sprite, move1, -1);
+      sprite.alternatingMoveInfo = {
+        start: currentMeasure,
+        cadence: n,
```

Comment:
Nit - n isn't a very descriptive variable name.

Reply:
`n` is used throughout the dance API to refer to measures, so I think it's okay to keep consistent with that pattern